data/permissions.ts

```diff
@@ -20,6 +20,9 @@ if (status.state === "granted") {
   console.log("'env' permission is denied.");
 }
 
+// There are also synchronous versions of all the permisison APIs
+Deno.permissions.requestSync({ name: "env" });
```

Comment:
I think we should assign the result to some variable. Otherwise it's a little unclear what's possible from this example